### DISCUSSION TITLE
Recon we could get a GameMaker Studio tile-based costumes?

### DISCUSSION THREAD

**Discussion by LeafiDev** [TurboWarp/extensions#893] [General]
For backgrounds to make it easier.
As an extension of course.

**Comment by LilyMakesThings** [comment]
You can replicate something like this with clipping and blending. I've neglected to try and make something with this but I might now.

**Reply by LeafiDev** [reply]
I was just experimenting with undertale-like concepts and wanted to do a cool background so I wondered if there was a extension anywhere for tile like backgrounds. As GameMaker Studio uses such a functionality

**Comment by snowboyz0825** [comment]
https://github.com/snowboyz0825/snowscodestuff/blob/main/SB3s/RPG%20Engine%20-%20Undertale%20Branch.sb3
Go crazy, I don't even think I know how this works and I made it
Also this
https://github.com/snowboyz0825/snowscodestuff/blob/main/SB3s/RPG%20Engine.sb3
Better commented, better coded, and better support in general. You may-or-may-not be able to just port the sprites from the first to the second. The only problem is the second one uses tile based movement, and if you've ever played an omori or something similair, basically that kind of movement. The first one has really, really bad support for "free movement," or movement that isn't locked to tiles. If you want to make tiles, you can use this 
https://www.piskelapp.com/p/create/sprite
Export as PNG, then upload it to turbowarp. Good luck, and if you actually use it in something big, gimme credit or money /hj
